addons/laser/XEH_postInit.sqf

```diff
@@ -33,7 +33,12 @@ if (hasInterface) then {
 ["ace_laserOn", {
     params ["_uuid", "_args"];
     TRACE_2("ace_laserOn eh",_uuid,_args);
+
     GVAR(laserEmitters) set [_uuid, _args];
+    private _unit = _args select 0;
+    if (hasPilotCamera _unit) then {
```

Comment:
don't we want some kind of locality check?

Reply:
`local _unit` should be safe enough, though this will require resetting the laser to work if locality changes.